### ISSUE TITLE
Errors when starting

### ISSUE THREAD

**Issue by dkruyt** [Drewster727/plexpy-influxdb-export#3]
After starting I get the following output, but nothing happens and nothing is getting into influxdb.

I am using the latest PlexPy from git.

Started
400: {"error":"error parsing query: found NOT, expected ; at line 1, char 20"}

'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'

**Comment by Drewster727** [comment]
Can you send over how you're calling the script? with args.
Haven't seen that myself. Perhaps you can hit the plexpy api in the browser directly and make sure that part is working as well.

**Comment by dkruyt** [comment]
API is working in the browser.

get_activity

{"response": {"message": null, "data": {"stream_count": "0", "sessions": []}, "result": "success"}}

also get_users is returning output.

calling as following
plexpy_influxdb_export.py --influxdbdatabase media --plexpyhost 192.168.XX.XX --plexpyapikey 9be1XXXXXXXXXXXXX9abb

Also have your sabnzbd_influxdb_export.py running with almost same arguments without problems.

**Comment by iswanb** [comment]
I had a similar problem on a Windows machine.

The "400" error is somewhere in the create_database function, but it isn't fatal. My script will throw the error then happily carry on. I think you may just need to manually create the database in InfluxDB.

On the 'unicode' error, looking at the code between the sabnzbd and Plexpy exporters, the `json.load` calls are different. I think the call should be `json.loads`

I changed the Plexpy code to:
`data = json.loads(req)` at line 19 and 
`data = json.loads(requests.get('{0}{1}'.format(plexpy_url, '&cmd=get_users'), verify=False).text)` at line 65

It worked like a charm. No more 'unicode' errors and data is flowing.

New to Github, so apologies if I'm posting wrong.

**Comment by thegreyfox** [comment]
Hi there,

I'm too getting the same error when starting it up 

`Started
400: {"error":"error parsing query: found NOT, expected ; at line 1, char 20"}

'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'
'unicode' object has no attribute 'read'`

Line 19 already shows as `data = json.loads(req)` and Line 65 shows up as `data = json.loads(requests.get('{0}{1}'.format(plexpy_url, '&cmd=get_users'), verify=False).text)`

**Comment by Drewster727** [comment]
Yea sorry guys, that was a change someone else made that I pushed in (without testing). I believe the .loads() call fixed it for them probably due to some oddball version of python they had. I will revert that.

**Comment by thegreyfox** [comment]
Thanks for looking into this, Drew!

**Comment by Drewster727** [comment]
@dkruyt @thegreyfox @iswanb 

guys, please post the following:
- OS (+ version)
- Python Version
- PlexPy Version

Thanks,
Drew

**Comment by thegreyfox** [comment]
@Drewster727 

Please see the details:

- OS : macOS 10.12.3
- Python: Python 2.7.10
- PlexPy: 1.4.16

**Comment by iswanb** [comment]
OS: Windows Server 2016 Essentials
Python: Python 2.7.12 (running from within Command Prompt)
PlexPy: 1.4.14 (guess I'm due for an update)

**Comment by dkruyt** [comment]
OS: Ubuntu 16.04.1 LTS
Python: 2.7.12
PlexPy: From Git

**Comment by dkruyt** [comment]
When I checkout 056cf25e3efb08385cd3a1b573fae626fe5e5979 it works

**Comment by thegreyfox** [comment]
I tried https://github.com/Drewster727/plexpy-influxdb-export/commit/056cf25e3efb08385cd3a1b573fae626fe5e5979 however python crashes.

**Comment by Drewster727** [comment]
Sorry guys, I haven't had a chance to look at this. Planning to soon. Thanks!

**Comment by JeordyR** [comment]
Hey Drewster, found your script while searching for a solution and decided to copy/rewrite it for the learning experience. While rewriting it I found the issues causing these reported errors:

- InfluxDB removed the 'CREATE DATABASE IF NOT EXISTS' option and set 'CREATE DATABASE' to do the same behavior by default (seems silly that they didn't just leave the old syntax as well to prevent this exact situation)
- json.load() does not work on string objects, requiring json.loads() to be used instead. I ended up just using the built-in json() function of requests. IE: data = requests.get(cmd, verify=False).json(). No clue what changed that broke this.

I would have modified and submitted a correction but I have no idea how GitHub works enough to do that yet :P. Can find mine here: https://github.com/JeordyR/Scripts/tree/master/plexpyInflux if you want to pull some stuff back out. I also added the pulling of libraries and their content counts as well.
